site/content/blog/2021-12-01-whats-new-in-svelte-december-2021.md

```diff
@@ -0,0 +1,91 @@
+---
+title: What's new in Svelte: December 2021
+description: "Svelte Summit Fall 2021 Recap, Rich Harris joins Vercel, SvelteKit opens GitHub Discussions"
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+With SvelteKit getting more and more stable each day, there's not much to cover in terms of code changes other than bug fixes... So, in this month's newsletter, we'll be covering Svelte Summit Fall 2021!
+
+If you want to dive deep into the last month's worth of bug fixes, check out the [Svelte](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md) and [SvelteKit](https://github.com/sveltejs/kit/blob/master/packages/kit/CHANGELOG.md) changelogs, respectively.
+
+
+## What happened at Svelte Summit?
+
+If you missed Svelte Summit, you can watch the entire live stream on [YouTube](https://www.youtube.com/watch?v=1Df-9EKvZr0) and catch a recap in the [#svelte-summit channel on Discord](https://discord.gg/YmHcdnhu).
+
+Here are the highlights:
```

Comment:
perhaps we could update each of these with a link directly to the talk? the talks are all listed here: https://www.youtube.com/channel/UCZSr5B0l07JXK2FIeWA0-jw

Reply:
Good idea, though I think a few talks still need to be uploaded.

Reply:
hmm. perhaps @kevmodrome has only listed some of them so far? but we could link to the main video (https://www.youtube.com/watch?v=1Df-9EKvZr0) with `#t=XmYs` in the URL (where X is number of minutes and Y is number of seconds)